packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -32,7 +39,31 @@ const Color _kDefaultGlowColor = Color(0xFFFFFFFF);
 @immutable
 class ScrollBehavior {
   /// Creates a description of how [Scrollable] widgets should behave.
-  const ScrollBehavior();
+  const ScrollBehavior({
+    @Deprecated(
+      'Temporary migration flag, do not use. '
+      'This feature was deprecated after v2.1.0-11.0.pre.'
+    )
+    bool useDecoration = false,
```

Comment:
Do you think we could avoid this flag altogether by making buildOverscrollIndicator just call buildViewportChrome by default?